### DISCUSSION TITLE
Description tooltips for fields

### DISCUSSION THREAD

**Discussion by iakshay** [payloadcms/payload#1740] [Feature Requests & Ideas]
Some advanced fields need more than just label to describe how they are used. Would be nice to add an optional field `tooltip` or `description` that shows a help icon next to the field label with more details.

Screenshot below from Woocommerce.

<img width="774" alt="image" src="https://user-images.githubusercontent.com/1104602/208955660-39430f85-641b-4ed0-a4ba-a678eea092f9.png">

**Comment by jacobsfletch** [comment]
@iakshay absolutely! Payload _does_ already have powerful field-level description. It accepts a string, function, or react component giving you full control over the UI. Check it out! https://payloadcms.com/docs/fields/overview#description

**Reply by iakshay** [reply]
Thanks, that works for my use case!
